src/DiskIO/DiskThreads/DiskThreadsDiskFile.cc

```diff
@@ -189,7 +189,7 @@ DiskThreadsDiskFile::close()
         ioRequestor->closeCompleted();
         return;
     } else {
-        debugs(79, DBG_CRITICAL, HERE << "DiskThreadsDiskFile::close: " <<
+        debugs(79, DBG_CRITICAL, "DiskThreadsDiskFile::close: " <<
```

Comment:
First of all, the script does not create blatantly malformed lines. It does not fix all the problems, but it does not create problems (bugs notwithstanding).

FWIW, I do not recall @kinkie presenting an equivalent solution. That does not mean it did not happen, of course! Please feel free to either encourage him to submit a competing PR or just use my old arguments against me in this PR if you think I was right back then. If I was right back then, then your (i.e., my old) arguments should win. If I was wrong back then, they should be defeated. We are here to select the best solution (among the available ones) and reject bad ones. It should not matter _who_ says something. Our decisions should be based on the _merits_.

Finally, please note that I started this PR with an explicit disclaimer that it is not clear whether doing these changes is a good idea. If we decide that it is better to keep the status quo, I am content with that. I offered a reasonable alternative so that we can make an informed choice and close this topic one way or the other.

P.S. Please, if at all possible, when you attribute some old actions or decisions to people, please include references. It is trivial to misinterpret something, and context matters.